src/EnergyPlus/SingleDuct.cc

```diff
@@ -4883,8 +4921,41 @@ namespace SingleDuct {
 		if ( BeginDayFlag ) {
 		}
 
-		if ( FirstHVACIteration ) {
+		if ( MyEnvrnFlag( ATMixerNum ) ) {
+			if ( ZoneEquipConfig( SysATMixer( ATMixerNum ).ZoneEqNum ).AirLoopNum > 0 ) {
+				SysATMixer( ATMixerNum ).AirLoopNum = ZoneEquipConfig( SysATMixer( ATMixerNum ).ZoneEqNum ).AirLoopNum;
+				MyEnvrnFlag( ATMixerNum ) = false;
```

Comment:
Seems like this would be a place to check if AirLoopNum == 0 then throw a fatal error, then the later if AirlloopNum > 0 tests can all be removed.